docs/UserGuide.md

```diff
@@ -167,6 +167,15 @@ Examples:
 - `find pm25`
 - `find singapore pm25`
 
+Description: Find API routes containing the search word in a specific field.
+
+Format: `find -x KEYWORD [MORE_KEYWORDS]…`, `find -t KEYWORD [MORE_KEYWORDS]…` (able to use any prefix)
+
+Examples:
+
+- `find -x get`
+- `find -t singapore pm25`
+
```

Comment:
Maybe instead of listing find command twice can differentiate the 2 ways to use it while listing it just once because on first glance it looked like a duplicate (similar to how run was listed).